debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -254,6 +256,18 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withValidation(Field::isNonNegativeInteger)
             .withDescription("This property can be used to reduce the connector memory usage footprint when changes are streamed from multiple tables per database.");
 
+    public static final Field MAX_RETRIABLE_RESTARTS = Field.create(MAX_RETRIABLE_RESTARTS_CONFIG_NAME)
```

Comment:
Please check `io.debezium.embedded.EmbeddedEngine.ERRORS_MAX_RETRIES`. We should unify it using the same option or option name.

Reply:
@jpechane I would have liked to initially. The reason I didn't is SQLServer connector has the concept of a streaming iteration and that gives us a good place to perform a reset of the retries. I'm not knowledgable enough about the other connectors to determine where a retries reset should occur in their respective logic. Perhaps we could set up a meeting to discuss it in more detail to see what could be done to accomplish this.